front_length: 0.6 
rear_length: 0.4